### PR TITLE
first code review

### PR DESCRIPTION
**Pull request by hiroto-udagawa**
looked through and made sure all the code ran and commented. still ned to brush up tsa_sciprt, time_shift_script, and noise_correction.

A few issues i found:
- Get rid of warnings
- We need to figure out the subject list indexing
- Should we make ‘make_convolve_lamdba’ into a function?
- Do we need to require stats models and patsy for ‘tsa_script'

### CODE REVIEW

**Comment by janewliang** [comment]
statsmodels (required for tsa_script) is already in requirements.txt. Where is patsy being imported? I don't see/recall it.

Really great work, though! Thanks so much.

**Comment by hiroto-udagawa** [comment]
patsy was called within statsmodel. I had to download both modules though. idk.. what do you think

**Comment by janewliang** [comment]
It's not that big of a deal to just add patsy, so I'll go ahead and do that on my next pull request to be safe.

File: code/utils/misc/BART_Data_Beginning.py [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+#  tree anatomy/
+#  3)
+#  tree behav/task001_run001
+#  4)
+#  tree model/model001/onsets/task001_run001
+#  5)
+#  tree model/model002/onsets/task001_run001
+
+# 1)
+BOLD/task001_run001
```
Comment (benjaminleroy) [inline code/utils/misc/BART_Data_Beginning.py]: This needs to be in quotes, as it will not run"

File: code/utils/misc/BART_Data_Beginning.py [hunk 2]
```diff
@@ -0,0 +1,296 @@
+
```
Comment (benjaminleroy) [inline code/utils/misc/BART_Data_Beginning.py]: Could you clean up the In[], etc.

File: code/utils/scripts/event_related_HRF_script.py [hunk 3]
```diff
@@ -264,7 +264,7 @@
 	# In [1]: %timeit testconv_np = np.convolve(neural_prediction, hrf_at_trs)
```
Comment (benjaminleroy) [inline code/utils/scripts/event_related_HRF_script.py:264]: Totally should run, but fine, so mean :'(

File: code/utils/scripts/hypothesis_script.py [hunk 4]
```diff
@@ -41,7 +38,7 @@
 cond3=np.loadtxt(condition_location+"cond003.txt")
 
 #######################
-# a. (my) convolution #
+# Ben's   convolution #
```
Comment (benjaminleroy) [inline code/utils/scripts/hypothesis_script.py]: Let's call this "smart" (I know they're the same ;P)

File: code/utils/scripts/lin_reg_plots.py [hunk 5]
```diff
@@ -23,19 +24,18 @@
 smooth_data           = final_data + 'smooth/'
 hrf_data              = final_data + 'hrf/'
 
-
-#sys.path.append(os.path.join(os.path.dirname(__file__), "../functions/"))
```
Comment (benjaminleroy) [inline code/utils/scripts/lin_reg_plots.py:27]: @janewliang 
Should we be doing the path append similar to line 27 or 28 for you (and linux users like Jarrod?)

File: code/utils/scripts/model_selection.py [hunk 6]
```diff
@@ -22,24 +28,24 @@
...
+#Choose which criteria
 input_var = input("adjR2 or BIC or AIC: ")
 
 
-################
-# Adjusted R^2 #
-################
+################################
+# Functions for different models #
+################################
```
Comment (benjaminleroy) [inline code/utils/scripts/model_selection.py:49]: The ### don't align :p

File: code/utils/scripts/lin_reg_plots.py (hunk 5 above)
Comment (janewliang) [inline code/utils/scripts/lin_reg_plots.py:27]: You don't have to. Something along the lines of "sys.path.append(location_of_functions)" where location_of_functions= ../functions or whatever has been working for me. I don't remember why I couldn't get it to cooperate earlier this semester, which is why a lot of older scripts have the os.path.join thing. "sys.path.append(location_of_functions)" is cleaner and easier, so I was going to update all the scripts with that later myself if you guys hadn't gotten to it already.

File: code/utils/scripts/model_selection.py (hunk 6 above)
Comment (hiroto-udagawa) [inline code/utils/scripts/model_selection.py:49]: thanks for pointing this out ben. how could i make such a mistake

**Comment by benjaminleroy** [comment]
Did you just clean up all my comments? I thought it would keep them :(, oh well. It's good right @janewliang

**Comment by janewliang** [comment]
Think it's good.

**Pull request merged**